src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -624,7 +624,7 @@ public boolean snapshotExists(Snapshot snapshot) throws InterruptedException {
     }
 
     @Override
-    public boolean deleteSnapshot(Snapshot snapshot) throws InterruptedException {
+    public boolean deleteSnapshot(Snapshot snapshot) throws InterruptedException { // TODO is this what should delete the views?
```

Comment:
The views themselves will go away when the snapshot BQ dataset goes away, since they are contained in the dataset. We don't have to delete each item in the BQ dataset.

The question I have is whether additional cleanup is needed, since the views are authorization views that have some permissions on the data in the BQ dataset(s). I didn't think so, but it would be good to be sure.